tool/autoqa/auto-annotate.js

```diff
@@ -68,19 +68,14 @@ module.exports = {
             default: null,
             help: `List of functions to include, split by comma (no space). Include all functions if not specified`,
         });
-        parser.add_argument('--skip', {
-            action: 'store_true',
-            help: 'Skip the entire process.',
-            default: false
-        });
         parser.add_argument('--remove-existing-canonicals', {
             action: 'store_true',
-            help: 'Remove all existing canonical annotations',
+            help: 'Remove all existing canonical annotations in the schema',
             default: false
         });
-        parser.add_argument('--algorithm', {
-            help: 'Different algorithms to generate canonicals including bert, bart, adj, domain_synonyms split by comma (no space)',
-            default: null
+        parser.add_argument('--algorithms', {
+            nargs: '*',
+            help: 'Different algorithms to generate canonicals including bert-property-synonyms, bart-paraphrase, bert-adjectives, bert-domain-synonyms'
         });
```

Comment:
What's the default here? Should we have a reasonable default? Is that BART only?

Reply:
The current default is nothing - we skip auto annotation, but yeah, I think we should set it to the one that we think is the best